src/shims/windows/foreign_items.rs

```diff
@@ -343,7 +343,9 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
             // Better error for attempts to create a thread
             "CreateThread" => {
                 this.check_abi(abi, Abi::System { unwind: false })?;
-                throw_unsup_format!("Miri does not support concurrency on Windows");
+
+                this.handle_unsupported("Miri does not support concurrency on Windows")?;
```

Comment:
```suggestion
                this.handle_unsupported("can't create threads on Windows")?;
```
That brings it more in line with the way the other errors are worded.